README.md

```diff
@@ -104,3 +104,18 @@ ansible-playbook ansible_playbook.yml -i ansible_inventory_file
 *when ansible playbook is run, it will prompt 'what is your api server token'
  please insert api server token, this is not purposefully migrated to host node for security 
 
+## How to build and run development and production containers
```

Comment:
These are really easy to follow (nice!) but they don't quite render correctly - [see what they look like here](https://github.com/NShilu/DevOps-Course-Starter/tree/Exercise-for-Module5)

Some suggestions:
* Always put a space after bullet points before the text, otherwise they render:
*like this
using backticks ( ` ) rather than quotes ( " ) will format your commands as code which can make them clearer